lib/anycable/nsq/handler.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+require 'anycable/socket'
+
+module AnyCable
+  module NSQ
+    class Handler
+      def handle(data)
+        request = JSON.parse(data)
+        case request['command']
+        when "connect"
+          connect(request)
+        when "disconnect"
+        when "perform", "subscribe", "unsubscribe"
+          perform(request)
+        end
+      end
+      # Handle connection request from WebSocket server
+      def connect(request)
+        logger.debug("RPC Connect: #{request}")
```

Comment:
is this something we want to leave in here long term,  or just while we're getting the rails upgrade up and running?

Reply:
Good question... These are just left over from Pete's work. 🤔 

We can prob delete or comment them out (I'm not even positive where these would show up).

Reply:
Actually, I'm going to leave these in. It looks like this is a pattern that the maintainer of the gem uses.